rustler/src/types/elixir_struct.rs

```diff
@@ -12,17 +12,15 @@ use super::map::map_new;
 use crate::{Env, NifResult, Term};
 
 pub fn get_ex_struct_name(map: Term) -> NifResult<Atom> {
-    let env = map.get_env();
     // In an Elixir struct the value in the __struct__ field is always an atom.
-    map.map_get(atom::__struct__().to_term(env))
-        .and_then(Atom::from_term)
+    map.map_get(atom::__struct__()).and_then(Atom::from_term)
 }
 
 pub fn make_ex_struct<'a>(env: Env<'a>, struct_module: &str) -> NifResult<Term<'a>> {
     let map = map_new(env);
 
-    let struct_atom = atom::__struct__().to_term(env);
```

Comment:
The PR would work without these changes as well, correct? But we can save some work as `to_term(env)` is essentially what happens inside of `map_put()` anyways? (I wonder if the PR is breaking our current APIs or not)